docs/source/whatsnew.rst

```diff
@@ -71,43 +75,36 @@ only when the channel application capability is updated to v2.0.
 See the :doc:`chaincode4noah` tutorial for complete details of the new
 chaincode lifecycle.
 
-External chaincode launcher
----------------------------
-
-While chaincode is still run in a docker container by default in Fabric v2.0,
-the external chaincode launcher feature empowers operators to build and launch
-chaincode with the technology of their choice.
-
-* **Eliminate Docker daemon dependency** Prior releases of Fabric required
-  peers to have access to a Docker daemon in order to build and launch
-  chaincode - something that may not be desirable in production environments
-  due to the privileges required by the peer process.
-
-* **Alternatives to containers** Chaincode is no longer required to be run
-  in Docker containers, and may be executed in the operator’s choice of
-  environment (including containers).
+New chaincode application patterns for collaboration and consensus
+------------------------------------------------------------------
 
-* **External builder executables** An operator can provide a set of external
-  builder executables to override how the peer builds and launches chaincode.
+The same decentralized methods of coming to agreement that underpin the
+new chaincode lifecycle management can also be used in your own chaincode
+applications to ensure organizations consent to data transactions before
+they are committed to the ledger.
 
-* **Chaincode as an external service** Traditionally, chaincodes are launched
-  by and then connect to the peer. It is now possible to run chaincode as
-  an external service, for example in a Kubernetes pod, which a peer can
-  connect to and utilize for chaincode execution. See :doc:`cc_service` for more
-  information.
+* **Automated checks** As mentioned above, organizations can add automated
+  checks to chaincode functions to validate additional information before
```

Comment:
Should have a period after this, other wise you'll have a bolded word followed by an unbolded word, which will look weird.

Reply:
You are technically correct Joe, although I think the doc reads fine because the bolded text emphasis is consistent throughout the doc.  If using bold emphasis on the text is meant to serve as a header to the sub-section, then perhaps instead of a period, the follow-on text should be underneath?  I am however, inclined to vote for leaving as-is.

Reply:
I hadn't noticed the other instances of this in the doc, but I would absolutely change those as well. 

![Screen Shot 2020-01-29 at 9 47 11 AM](https://user-images.githubusercontent.com/26771644/73366708-65d42000-427c-11ea-8ba2-631d6b5bcaf2.png)

Looks wrong. Is wrong. Imo.

Reply:
I don't mind the option of having the text underneath, either.

Reply:
I viewed the entire doc in html.  I suggest then we change all from:
**Bold text** Follow-on text...
To:
**Bold Text**
Follow-on-text...

Reply:
For consistency, we should probably add a 'style guide' topic to the doc working group meeting agenda.

Reply:
I had tried @denali49 suggestion in the rst source, but it ends up rendered the same. I'm going to leave it as-is for now, the doc masters can work the magic in a follow-on PR if it doesn't look right.